tools/build-image.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+LC_ALL=C.UTF-8
```

Comment:
This looks familiar to me, when I use tools on MacOS and trying to ssh to any linux box.